BedrockServer.cpp

```diff
@@ -10,10 +10,14 @@ set<string>BedrockServer::_blacklistedParallelCommands;
 shared_timed_mutex BedrockServer::_blacklistedParallelCommandMutex;
 
 void BedrockServer::acceptCommand(SQLiteCommand&& command, bool isNew) {
+    acceptCommand(make_unique<SQLiteCommand>(move(command)), isNew);
+}
+
```

Comment:
I've added the above version of this just to avoid having a co-requisite auth change. Once this change goes out, we can change auth to use the below version, and remove the above version.

Auth doesn't actually send `AuthCommand` objects here currently anyway, it just sends `SQLiteCommand` wrappers around requests. We could change it to send actual `AuthCommand` objects if we like, but it doesn't much matter for this particular case.